python/postprocessing/framework/crabhelper.py

```diff
@@ -5,62 +5,63 @@
 import re
 import PSet
 
+
 def inputFiles():
-   print "ARGV:",sys.argv
-   JobNumber=sys.argv[1]
-   crabFiles=PSet.process.source.fileNames
-   print crabFiles
-   firstInput = crabFiles[0]
-   tested=False
-   forceaaa=False
-   print "--------------- using edmFileUtil to convert PFN to LFN -------------------------"
-   for i in xrange(0,len(crabFiles)) :
-     if os.getenv("GLIDECLIENT_Group","") != "overflow" and  os.getenv("GLIDECLIENT_Group","") != "overflow_conservative" and not forceaaa:
-       print "Data is local"
-       pfn=os.popen("edmFileUtil -d %s"%(crabFiles[i])).read()
-       pfn=re.sub("\n","",pfn)
-       print crabFiles[i],"->",pfn
-       if not tested:
-         print "Testing file open"
-         import ROOT
-         testfile=ROOT.TFile.Open(pfn)
-         if testfile and testfile.IsOpen() :
-            print "Test OK"
-            crabFiles[i]=pfn
-            testfile.Close()
-            #tested=True
-         else :
-            print "Test open failed, forcing AAA"
-            crabFiles[i]="root://cms-xrd-global.cern.ch/"+crabFiles[i]
-            forceaaa=True
-       else :
-            crabFiles[i]=pfn
+    print("ARGV: " + sys.argv)
```

Comment:
This does not work in python 2

Reply:
Sorry about that, indeed str() is missing.